api/tests/setup/accounts/creation_helpers.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+def create_user(email: Optional[str] = None, password: Optional[str] = None):
+    email = email or "test@example.com"
+    password = password or "Password123!"
+    user = User.objects.create(
+        first_name="Test",
+        last_name="User",
+        email=email,
+        username=email,
+    )
+    user.set_password(password)
+    user.save()
+    return user
```

Comment:
In keeping with reducing repetition, it's a good idea to create helpers to create things for your tests if they are used frequently